charts/portworx/values.yaml

```diff
@@ -27,9 +27,9 @@ customRegistryURL:
 registrySecret:
 
 lighthouse: true
-lighthouseVersion: 2.0.4
-lighthouseSyncVersion: 0.3
-lighthouseStorkConnectorVersion: 0.1
+lighthouseVersion: 2.0.5
+lighthouseSyncVersion: 2.0.5
```

Comment:
did we sync all these versions to 2.0.5?

Reply:
https://install.portworx.com/2.1 is using

portworx/px-lighthouse:2.0.4
portworx/lh-config-sync:0.4
portworx/lh-stork-connector:0.2

Helm should be at the same versions

Reply:
We need 2.0.5. for a fix regarding disks. there is no difference between the other pieces minus that. we're just pulling versions in line. If we need to update install.portworx.com then so be it.

Reply:
We started syncing them for the operator.
Next step is just having 1 version in this values file and using that for all the images.

Reply:
Okay. 

Please send the px-installer PR too in that case. They need to be in sync.